Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -954,6 +954,22 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Verify if string matches the regex pattern and return corresponding parts of string.
+    ///
+    /// - Parameter pattern: Pattern to verify.
+    /// - Returns: Matching pattern parts of string.
+    /// - Throws: Throws any errors thrown by creation of invalid pattern.
+    func matchings(pattern: String) throws -> [String] {
```

Comment:
Thanks for the extension @DimaZava, Small comment, I think the signature `matches(for pattern: String)` makes more sense here

Reply:
+1